src/components/MermaidDash.js

```diff
@@ -163,8 +164,8 @@ class MermaidDash extends Component {
         this.setState({ metrics });
       }
 
-      if (prevMetricUsersCount !== this.getCount(updatedSites, 'project_admins')) {
-        metrics[2].count = this.getCount(updatedSites, 'project_admins');
+      if (prevMetricUsersCount !== this.getUserCount(updatedSites, projectChoices)) {
```

Comment:
you should only do this count once and save to variable instead of counting twice

`this.getUserCount(updatedSites, projectChoices)`